.env.tpl

```diff
@@ -1,5 +1,11 @@
-LOGS_TOKEN=
+NODE_ENV=dev
+
+# logging
 BUGS_TOKEN=
+BUGSNAG_RELEASE_STAGES=production,staging
+LOGSTASH_HOST=logstash.dialonce.io
+LOGSTASH_PORT=5000
```

Comment:
Please remove those values, this is a public project...

Reply:
:anguished: done